mautic/mautic
# WIP: Creates UI config field for `theme_email_default` in `Configuration > Theme Settings > Default theme`

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12807  



#### Description:

Introduces a UI field to set the `theme_email_default` config key, enhancing #9189 and (partly) fixing #12807.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open `Configuration > Theme Settings > Default theme`
3. Set a new default email builder theme.

## Help wanted

1. If you click save, Mautic won't save the new setting properly. I don't know why.
2. I wanted to make a visual separation between the form/page and the email theme. I wanted to use sections like in `Configuration > System Settings`, but I don't know how.
